src/main/java/training/subsystems/RobotArm/Roller.java

```diff
@@ -0,0 +1,71 @@
+package training.subsystems.RobotArm;
+
+import com.revrobotics.CANSparkBase;
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.geometry.Rotation2d;
+import utils.GBSubsystem;
+
+public class Roller extends GBSubsystem {
+
+	private CANSparkMax motor;
+	private static Roller instance;
+
+	private Roller() {
+		this.motor = new CANSparkMax(RolerConstens.MOTOR_ID, CANSparkLowLevel.MotorType.kBrushless);
+		motor.getPIDController().setP(RolerConstens.KP);
+		motor.getPIDController().setI(RolerConstens.KI);
+		motor.getPIDController().setD(RolerConstens.KD);
+	}
+
+	public static Roller getInstance() {
+		if (instance != null)
+			instance = new Roller();
+		return instance;
+	}
+
+	public void moveAtSpeed(Rotation2d velocity) {
+		motor.getPIDController()
+			.setReference(
+				velocity,
+				CANSparkBase.ControlType.kVelocity,
+				RolerConstens.PID_SLOT,
+				RolerConstens.ARB_FEED_FOWORDS
+			);
+	}
+
```

Comment:
רקצםהק םמק ךןמק דפשבק good luck translating that

Reply:
remove one line space

Reply:
so remove it